ProcessMaker/bpm
# As a user without permission to cancel should not see the cancel button

fixes #976 

To test: 
log in as a user without permission to cancel a request, navigate to the request summary page and see that there is no long a section for "cancel"

There has also been unused code removed in the ProcessRequestController
![screen shot 2018-12-04 at 3 32 46 pm](https://user-images.githubusercontent.com/29641725/49480326-27a7b180-f7db-11e8-95be-4d7b40dc3c33.png)
